app.js

```diff
@@ -49,18 +87,10 @@ async function formSubmit(event) {
 
     console.info("Response", response);
     if (response.status === 200) {
+      const contentType = parseContentType(response);
```

Comment:
As above, pass resonse.headers.get("content-type") and deal with undefined/empty in the function itself.